kernel/common/ezautolinkoperator.php

```diff
@@ -59,20 +108,25 @@ function addURILinks( $text, $max, $methods = 'http|https|ftp' )
     function modify( $tpl, $operatorName, $operatorParameters, $rootNamespace, $currentNamespace, &$operatorValue, $namedParameters, $placement )
     {
         $ini = $tpl->ini();
-        $max = $ini->variable( 'AutoLinkOperator', 'MaxCharacters' );
+        self::$max = $ini->variable( 'AutoLinkOperator', 'MaxCharacters' );
         if ( $namedParameters['max_chars'] !== null )
         {
-            $max = $namedParameters['max_chars'];
+            self::$max = $namedParameters['max_chars'];
         }
 
         $methods = $ini->variable( 'AutoLinkOperator', 'Methods' );
         $methodText = implode( '|', $methods );
 
+        self::$isLinkWwwUrl = ( $ini->variable( 'AutoLinkOperator', 'LinkWwwUrlsWithoutProtocol' ) == 'enabled' );
+        
+        $hideProtocls = $ini->variable( 'AutoLinkOperator', 'HideProtocols' );
```

Comment:
This variable name should also be changed to `$stripProtocolsText` at the very least. We think this was merely a spelling typo. We request you change this variable name.